client/js/components/tags/TagsList.vue

```diff
@@ -201,16 +121,6 @@ export default {
       return Object.keys(this.tagAttributes)
```

Comment:
I wonder if we want to keep it like this, or just type the two keys in the Request "hard coded"

Reply:
hm, no i think its unnecessarily and could be refactored: https://github.com/demos-europe/demosplan-core/pull/4256/commits/a66f487c978895bde9aaa4336418b9f8781d59ec. would this be ok?